chromium_src/BUILD.gn

```diff
@@ -87,7 +90,10 @@ static_library("chrome") {
   }
 
   if (is_linux) {
-    sources += [ "//chrome/browser/media/webrtc/window_icon_util_linux.cc" ]
+    sources += [
+      "//chrome/browser/media/webrtc/window_icon_util_linux.cc",
+      "//chrome/browser/process_singleton_posix.cc",
```

Comment:
Can you move this under `is_posix`, also looks like we need `process_singleton_mac.mm` for https://chromium-review.googlesource.com/c/chromium/src/+/801030/ . Can you confirm if that is the case ?

Reply:
Looks like adding `process_singleton_mac.mm` can help with an edge case involving "open URL" events https://chromium-review.googlesource.com/c/chromium/src/+/801030, though I'm not sure how applicable it is to Electron apps.
The linked changeset also adds a line to `app_controller_mac.mm`, though that line isn't there in the current Chromium code.